openapi/components/schemas/Plans/FlexiblePlan.yaml

```
@@ -3,6 +3,6 @@ allOf:
   - $ref: ./Plan.yaml
 properties:
   id:
-    description: ID of a plan you wish to modify.
+    description: ID of a plan you want to modify.
```

Suggestion: I think we can leave the second part of this sentence, for consistency. 
```suggestion
    description: ID of the plan.
```